src/main/java/com/vaadin/guice/server/GuiceNavigator.java

```diff
@@ -0,0 +1,25 @@
+package com.vaadin.guice.server;
+
+import com.vaadin.navigator.Navigator;
+import com.vaadin.navigator.ViewDisplay;
+import com.vaadin.ui.ComponentContainer;
+import com.vaadin.ui.SingleComponentContainer;
+import com.vaadin.ui.UI;
+
+public class GuiceNavigator extends Navigator {
```

Comment:
a little javadoc would be nice here, like 'subclass this class if you want custom navigation behaviour'